pkgs/development/compilers/scopes/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, fetchFromBitbucket
+, llvm
+, clang
+, clang-unwrapped
+, genie
+, spirv-tools
+, spirv-cross
+, libc-hdrs
+, clang-hdrs
+}:
+stdenv.mkDerivation rec {
+  pname = "scopes";
```

Comment:
Can you apply a similar style you've used in `genie` in `lua-packages.nix`?
When attributes are separated in that way it's much easier to read.

Reply:
Done.